src/internal/util/exportPattern.ts

```diff
@@ -2729,7 +2729,7 @@ export function includesLanguagePlaceholders(str: string): boolean {
 }
 
 export function replaceLanguagePlaceholders(str: string, languageCode: string): string {
-    const language = languages.find(l => l.twoLettersCode === languageCode);
+    const language = languages.find(l => l.twoLettersCode === languageCode || l.locale === languageCode);
```

Comment:
Could you please also add a test for this change? There is already a set of several cases in test suite and would be nice to  see additional case where this change comes into action. Thanks

Reply:
Added tests using the locale